### DISCUSSION TITLE
Reference the CN in the CAN

### DISCUSSION THREAD

**Discussion by PaulGambke** [OP-TED/eForms-SDK#635] [Q&A] [answered]
Hi,

which field do I need to use to reference the CN in the CAN?
Or is it not necessary anymore to do that?

**Comment by YvesJo** [comment, marked as answer]
Hi,
A Competition and a Result Notice will share the same Procedure ID if they belong to a same procedure, therefore there is no need to refer to the Competition notice from the Result notice.
The only cases where a reference to a specific notice is expected are:

- identification of the notice object to a Change with _Change Notice Version Identifier (BT-758)_,
- identification of the notice containing the contract subject to a Modification with _Modification Previous Notice Section Identifier (BT-1501)_,
- identification of the PIN only notice whose Parts contributed to the definition of the Lot with _Previous Planning Identifier (BT-125)_, 
- identification of the notice that announced the Framwork Contract used for the current contract with _Framework Notice Identifier (OPT-100)_, and
- identification of the previous notice which wa a TEDXML and does not therefore contain a Procedure ID using _Previous Notice (OPP-090)_

Also for non-technical questions preferably contact the Helpdesk directly [ted@publications.europa.eu](mailto:ted@publications.europa.eu).

**Comment by PaulGambke** [comment]
ok perfect, I will do that in the future. Thank you for the quick answer
